src/core.c

```diff
@@ -499,17 +499,33 @@ void InitWindow(int width, int height, void *data)
 #endif
 
 #if defined(PLATFORM_ANDROID)
+/* To allow easier porting to android, we allow the user to define a main function
+ * which we call from android_main, that we define ourselves
+ */
+extern int main(int argc, char *argv[]);
+void android_main(struct android_app *_app) {
+    char arg0[] = "raylib"; /* argv[] are mutable */
+    app = _app;
+    /* TODO should we maybe report != 0 return codes somewhere? */
+    (void)main(1, (char*[]){ arg0, NULL });
+}
+/* TODO add this to header, if apps really need it) */
```

Comment:
Do they need it?

Reply:
Actually, user doesn't need it, android app lifecycle is [managed internally by raylib](https://github.com/raysan5/raylib/blob/master/src/core.c#L2902).

Reply:
[android_native_app_glue.h](https://github.com/raysan5/raylib/blob/master/src/external/android/native_app_glue/android_native_app_glue.h#L111) has more info about that structure.